lib/translator/translation.cc

```diff
@@ -129,6 +130,30 @@ BeginResponse Translation::Begin(Span<const uint8_t> scsi_cmd,
       pipeline_status_ = VerifyToNvme(scsi_cmd_no_op, nvme_wrappers_[0]);
       nvme_cmd_count_ = 1;
       break;
+    case scsi::OpCode::kWrite6:
+      pipeline_status_ =
+          Write6ToNvme(scsi_cmd, nvme_wrappers_[0], allocations_[0], nsid,
```

Comment:
also this should be passing in `scsi_cmd_no_op` for the command with the opcode truncated off. Otherwise, it won't cast correctly

Reply:
Don't think this was fixed